content/W2017/W17-event-dinner-with-professors.md

```diff
@@ -0,0 +1,20 @@
+title: Dinner with the Professors
+Date: 2017-03-1
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Student-faculty dinner.
```

Comment:
This should be something like "Join WiCS for our second dinner with profs event!" or something to that effect. i.e more pep